kernel/meson.build

```diff
@@ -378,22 +395,22 @@ base_kops = [
   },
   { 'base': '?geru',
     'modes': {
-        'c': {'exts': {'_k': {'addl': ['-UDOUBLE', '-DCOMPLEX', '-UCONJ']}}},
-        'z': {'exts': {'_k': {'addl': ['-DDOUBLE', '-DCOMPLEX', '-UCONJ']}}},
```

Comment:
@HaoZeke We remove all `-UDOUBLE` and `-DCOMPLEX` because they're added anyway based on `'c'`/`'z'` symbol, right?

Reply:
Yup. Probably in a later refactor some more of these can be abstracted away as well, like the `3m_startswith` and `endswith` rules.